src/platform/graphics/webgpu/webgpu-render-target.js

```diff
@@ -12,6 +12,8 @@ const stringIds = new StringIds();
 
 /**
  * Private class storing info about color buffer.
+ *
+ * @private
```

Comment:
I'm guessing this is unnecessary because the root `index.js` does not export this class.

Reply:
yeah but I had to add it to avoid the type errors coming from WebGPU types